WalletWasabi.Tests/RegTests.cs

```diff
@@ -2664,7 +2664,7 @@ private void WalletTestsAsync_MemPoolService_TransactionReceived(object sender,
 				}
 
 				var times = 0;
-				while (wallet.Coins.Where(x => x.Label == "ZeroLink Change" && x.Unspent).SingleOrDefault() == null)
+				while (wallet.Coins.Where(x => x.Label == "ZeroLink Change" && x.Unspent).FirstOrDefault() == null)
```

Comment:
`x.Label == "ZeroLink Change" && x.Unspent` is not inherently unique.

Reply:
Improving the tests performance is not very important. Anyway, the equivalent statement is:
```c#
while (!wallet.Coins.Any(x => x.Unspent && x.Label == "ZeroLink Change" ))
```

Reply:
Reverted.